_posts/2023-04-21-tensorflow-mnist-classification.adoc

```diff
@@ -0,0 +1,405 @@
+---
+layout: post
+title:  Image classification with Debezium and TensorFlow
+date:   2023-04-21 10:20:00 -0100
+tags: [ machine-learning, tensorflow, examples, apache-kafka ]
+featured: true
+author: vjuranek
+---
+
+With the recent success of ChatGTP we can observe another wave of interest in the AI field and in machine learning in general.
+The previous wave of interest in this field was, at least to a certain extent, caused by the fact that awesome ML frameworks like https://www.tensorflow.org/[TensorFlow], https://pytorch.org/[PyTorch] or general data processing frameworks like https://spark.apache.org/[Spark] became available and made writing of ML models much more easy.
+Since that time these frameworks have matured and writing models is even easier, as you will see later in this blog.
+However, data set preparation and gathering data from various sources can be sometimes hard or even painful.
+Creating a complete pipeline which would pull existing or newly created data, adjust the data and ingest it into selected ML libraries can be quite a challenging task.
+Let's try to investigate if Debezium can help with this task and explore how we can leverage Debezium capabilities to make it more easy.
```

Comment:
```suggestion
With the recent success of ChatGPT, we can observe another wave of interest in the AI field and machine learning in general.
The previous wave of interest in this field was, at least to a certain extent, caused by the fact that excellent ML frameworks like https://www.tensorflow.org/[TensorFlow], https://pytorch.org/[PyTorch] or general data processing frameworks like https://spark.apache.org/[Spark] became available and made the writing of ML models much more straightforward.
Since that time, these frameworks have matured, and writing models are even more accessible, as you will see later in this blog.
However, data set preparation and gathering data from various sources can sometimes take time and effort.
Creating a complete pipeline that would pull existing or newly created data, adjust it, and ingest it into selected ML libraries can be challenging.
Let's investigate if Debezium can help with this task and explore how we can leverage Debezium's capabilities to make it easier.
```